addons/tm_tmf_loadouts/CfgLoadouts.hpp

```diff
@@ -716,6 +716,12 @@ class CfgLoadouts
         category = "Other";
         #include "loadouts\afr_militia.hpp"
     };
+    class AFR_AR
+    {
+        displayName = "Generic African Conventional Army (1990-)";
```

Comment:
`African Army (1990-)` would sort better with the other african loadout

Reply:
Will fix